modules/tests/shared/src/test/scala/gsp/math/skycalc/solver/ConstraintSpec.scala

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gsp.math.skycalc.solver
+
+import cats.tests.CatsSuite
+import gsp.math.skycalc.SkyCalcResults
+import java.time.Instant
+import java.time.Duration
+import gsp.math.Declination
+import implicits._
```

Comment:
It is hard To tell to which package this belongs to

Reply:
Agreed. Let's make the implicits import explicit.